core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/huggingFace/HuggingFaceSentimentAnalysisOpDesc.scala

```diff
@@ -0,0 +1,64 @@
+package edu.uci.ics.texera.workflow.operators.huggingFace
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, Schema}
+
+class HuggingFaceSentimentAnalysisOpDesc extends PythonOperatorDescriptor {
+  @JsonProperty(value = "attribute", required = true)
+  @JsonPropertyDescription("column to perform sentiment analysis on")
+  @AutofillAttributeName
+  var attribute: String = _
+
+  override def generatePythonCode(): String = {
+    s"""from pytexera import *
+       |from transformers import pipeline
+       |from transformers import AutoModelForSequenceClassification
+       |from transformers import TFAutoModelForSequenceClassification
+       |from transformers import AutoTokenizer, AutoConfig
+       |import numpy as np
+       |from scipy.special import softmax
+       |
+       |class ProcessTupleOperator(UDFOperatorV2):
+       |
+       |    def open(self):
+       |        model_name = "cardiffnlp/twitter-roberta-base-sentiment-latest"
+       |        self.tokenizer = AutoTokenizer.from_pretrained(model_name)
+       |        self.config = AutoConfig.from_pretrained(model_name)
+       |        self.model = AutoModelForSequenceClassification.from_pretrained(model_name)
+       |
+       |    @overrides
+       |    def process_tuple(self, tuple_: Tuple, port: int) -> Iterator[Optional[TupleLike]]:
+       |        encoded_input = self.tokenizer(tuple_["text"], return_tensors='pt')
```

Comment:
you need to let the user to choose the field, instead of hard coding it to "text" field.

Reply:
check the existing "Sentiment Analysis" interface and mimic from there.